fdmprinting/archive_retrieve.py

```diff
@@ -58,6 +60,7 @@ def save_image(printer):
         print("Failed to capture image from " + printer["ip"] + " Error:" + e)
```

Comment:
my bug, but since it's an easy fix, let's fix it here: this concats mixed types, so if you use commas instead, native types will call `toStr()` (or what ever):
```suggestion
        print("Failed to capture image from ", printer["ip"], " Error:", e)
```